mod_reforged/hooks/config/strings.nut

```diff
@@ -844,12 +844,12 @@ foreach (vanillaDesc in vanillaDescriptions)
  	}),
  	RF_Bloodlust = ::UPD.getDescription({
  		Fluff = "When surrounded by carnage, you feel revitalized and right at home!",
- 		Requirement = "Melee Attack",
+ 		Requirement = "Cleaver",
  		Effects = [{
 			Type = ::UPD.EffectType.Passive,
 			Description = [
-				"During your [turn|Concept.Turn], every successful attack reduces current [Fatigue|Concept.Fatigue] by " + ::MSU.Text.colorGreen("5%") + " per stack of [Bleeding|Skill+bleeding_effect] on the target and increases [Fatigue Recovery|Concept.FatigueRecovery] by " + ::MSU.Text.colorGreen("+1") + " for one [turn|Concept.Turn] per stack of [Bleeding|Skill+bleeding_effect] on the target.",
-				"[Bleeding|Skill+bleeding_effect] inflicted by the attack, or killing a target, also counts towards the bonus."
+				"After inflicting a fatality, gain " + ::MSU.Text.colorGreen("2") + " stacks of Bloodlust. For each stack [Resolve|Concept.Bravery] and [Initiative|Concept.Initiative] are increased by " + ::MSU.Text.colorGreen("25%") + " and skills build up " + ::MSU.Text.colorGreen("25%") + " less [Fatigue|Concept.Fatigue], stacking [additively|Concept.StackAdditively].",
```

Comment:
This should say somewhere more explicitly and earlier, that the maximum amount of stacks is 2. That information is currently only in the second half of the second bullet point where it looks lost and comes a bit too late.

I would use three bullet points in total:
- How are stacks gained / Maximum Stacks
- Effect per stack
- How are stacks lost/removed?